### ISSUE TITLE
WCAG AA level - Multiple ways

### ISSUE THREAD

**Issue by TeBenachi** [WordPress/twentytwentytwo#83]
A site must have multiple ways to find a page on the website. I think the only way to meet this requirement as a FSE is to add the search in the header or the footer. 

https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=245#multiple-ways

**Comment by kjellr** [comment]
I'm curious how this requirement was met with Twenty Twenty-One last year? That theme targeted AAA level, but did not include a search field by default. 

I think that this could be fulfilled many different ways: two navigation areas, a sitemap, etc. It's partially a theme problem, but also a consideration for content creators (especially in the case of a block theme).

**Comment by colorful-tones** [comment]
Additional nuance and context should be considered here. Yes, this theme has the outright potential to be used on 10,000+ sites, but it is hard to speculate whether those sites will have 2 pages, or 30+ pages.

From [Understanding Success Criterion 2.4.5: Multiple Ways](https://www.w3.org/WAI/WCAG21/Understanding/multiple-ways.html) > [Intent](https://www.w3.org/WAI/WCAG21/Understanding/multiple-ways.html#intent)

> The intent of this Success Criterion is to make it possible for users to locate content in a manner that best meets their needs. Users may find one technique easier or more comprehensible to use than another.

> Even small sites should provide users some means of orientation. For a three or four page site, with all pages linked from the home page, it may be sufficient simply to provide links from and to the home page where the links on the home page can also serve as a site map.

Furthermore, one of the Sufficient Techniques is: [G185: Linking to all of the pages on the site from the home page](https://www.w3.org/WAI/WCAG21/Techniques/general/G185) would assume that incorporating navigation to all pages in the site would allow coverage in a minimum form. 🤔

**Comment by TeBenachi** [comment]
Thank you for pointing out the [G185: Linking to all of the pages](https://www.w3.org/WAI/WCAG21/Techniques/general/G185). Considering the homepage can serve the purpose,  I think it’s safe to say no need for the search as default.
